pkg/interface/src/views/landscape/components/ResourceSkeleton.tsx

```diff
@@ -12,7 +12,7 @@ import GlobalApi from '~/logic/api/global';
 import { isWriter } from '~/logic/lib/group';
 import { getItemTitle } from '~/logic/lib/util';
 
-const TruncatedBox = styled(Box)`
+const TruncatedText = styled(RichText)`
   white-space: pre;
   text-overflow: ellipsis;
```

Comment:
wonder if setting
```css
a {
  max-width: 100px;
}
```
would be worthwhile?

Reply:
? Why? That seems trickier than it’s worth. The thing here is that truncating the box won’t properly flex the text dynamically. If we let the flex go up each container then we don’t need fixed pixel widths. It’s much more fluid.